sdk/rust/oak_tests/src/lib.rs

```diff
@@ -548,7 +565,7 @@ pub unsafe extern "C" fn wait_on_channels(buf: *mut u8, count: u32) -> u32 {
         for (i, handle) in handles.iter().enumerate() {
             let channel_status = RUNTIME
```

Comment:
Could `RUNTIME.read.expect(...)` be encapsulated in a top-level `fn` that always returns the value or panics?

Reply:
I played with that, but it doesn't really work because the lifetime of the returned reference isn't tightly scoped enough.